cups/ppd.c

```diff
@@ -1202,27 +1204,32 @@ _ppdOpen(
 	    sptr ++;
 	}
 
-      ppd->num_emulations = count;
-      if ((ppd->emulations = calloc((size_t)count, sizeof(ppd_emul_t))) == NULL)
+      temp_ptr = realloc(ppd->emulations, (size_t)(ppd->num_emulations+count)*sizeof(ppd_emul_t));
```

Comment:
Sadly, CUPS runs on platforms where `realloc(NULL, size)` will crash hard. But I can refactor this.